hack/make/.ensure-syscall-test

```diff
@@ -4,5 +4,20 @@ set -e
 # Build a C binary for cloning a userns for seccomp tests
 # and compile it for target daemon
 if [ "$DOCKER_ENGINE_GOOS" = "linux" ]; then
-	docker build ${DOCKER_BUILD_ARGS} -qt syscall-test contrib/syscall-test > /dev/null
+	if [ "$DOCKER_ENGINE_GOARCH" = "$DOCKER_CLIENT_OSARCH" ]; then
```

Comment:
That comment is correct in both places; "OSARCH" is the full "os/arch" string, while "GOARCH" (referenced here) is only the "arch" half, so mixing the two forms doesn't make a lot of sense.

So this line should be `if [ "$DOCKER_ENGINE_OSARCH" = "$DOCKER_CLIENT_OSARCH" ]; then` (as in my previous comment).